src/Workspaces/Core/Portable/CodeFixesAndRefactorings/DocumentBasedFixAllProviderHelpers.cs

```diff
@@ -119,8 +119,7 @@ internal static class DocumentBasedFixAllProviderHelpers
         //
         // Do this in parallel across all the documents that were fixed.
 
-        var finalSolution = currentSolution;
```

Comment:
no need for a capture here anymore.  consumeItems can now return the final transformed value.

Reply:
nit: can you mark the lambda as static to make clear it's not capturing

Reply:
Will do!